### PR TITLE
fix_sequence_conv_op

### PR DESCRIPTION
**Pull request by chengduoZH**
refine SequenceConvOp
continue https://github.com/PaddlePaddle/Paddle/pull/4814

### CODE REVIEW

File: paddle/operators/sequence_conv_op.cc [hunk 1]
```diff
@@ -128,25 +128,25 @@ class SequenceConvOpMaker : public framework::OpProtoAndCheckerMaker {
         "this LoDTensor is a matrix with shape (T, D), where, T is the "
         "total time steps in this mini-batch, D is the output feature size.");
 
-    AddAttr<bool>("padding_trainable",
+    AddAttr<bool>("paddingTrainable",
```
Comment (qingqing01) [inline paddle/operators/sequence_conv_op.cc]: Line 120 - 123, the shape for `Filter` is not right.

File: paddle/operators/sequence_conv_op.cc [hunk 2]
```diff
@@ -128,25 +128,25 @@ class SequenceConvOpMaker : public framework::OpProtoAndCheckerMaker {
...
+    AddAttr<int>("contextStart",
+                 "(int, default 0) the contextStart of SequenceConvOp "
                  "represents the beginning of the convolution of the number of "
                  "rows of sequence, which can be negative.")
         .SetDefault(0);
-    AddAttr<int>("context_stride",
-                 "(int, default 1) the context_stride of SequenceConvOp "
+    AddAttr<int>("contextStride",
+                 "(int, default 1) the contextStride of SequenceConvOp "
                  "represents the step length of convolution. "
```
Comment (qingqing01) [inline paddle/operators/sequence_conv_op.cc]: step length convolution -> stride length of convolution kernel.

File: paddle/operators/sequence_conv_op.cc [hunk 3]
```diff
@@ -128,25 +128,25 @@ class SequenceConvOpMaker : public framework::OpProtoAndCheckerMaker {
...
+                 "(int, default 3) the contextLength of SequenceConvOp is the "
                  "height of the convolution kernel.")
         .SetDefault(3)
         .GreaterThan(0);
-    AddAttr<int>("context_start",
-                 "(int, default 0) the context_start of SequenceConvOp "
+    AddAttr<int>("contextStart",
+                 "(int, default 0) the contextStart of SequenceConvOp "
                  "represents the beginning of the convolution of the number of "
                  "rows of sequence, which can be negative.")
```
Comment (qingqing01) [inline paddle/operators/sequence_conv_op.cc]: More explanations:

The negative number means to pad contextStart time-steps of zeros or learnable parameters at the beginning of each instance. The positive number means to skip contextStart time-steps of each instance.

File: paddle/operators/sequence_conv_op.cc [hunk 4]
```diff
@@ -128,25 +128,25 @@ class SequenceConvOpMaker : public framework::OpProtoAndCheckerMaker {
...
+    AddAttr<bool>("paddingTrainable",
                   "(bool, default false) the padding data of SequenceConvOp "
                   "is trainable or not.")
         .SetDefault(false);
-    AddAttr<int>("context_length",
-                 "(int, default 3) the context_length of SequenceConvOp is the "
+    AddAttr<int>("contextLength",
+                 "(int, default 3) the contextLength of SequenceConvOp is the "
                  "height of the convolution kernel.")
         .SetDefault(3)
```
Comment (qingqing01) [inline paddle/operators/sequence_conv_op.cc]: I think this is an argument the users must set, so please do not set the default value.

**Comment by chengduoZH** [comment]
From the realization of function, using [StridedCopy](https://github.com/PaddlePaddle/Paddle/pull/4205/files) in context_project is also appropriate. But from the efficiency analysis, the efficiency of using StridedCopy is not high.
For example, the length of sequence is 20, context_length is 3, the realization of StridedCopy is equivalent to the replication of the 20*3 times, and for col_matrix, the address of each time replication is discontinuous.

**Comment by chengduoZH** [comment]
From the realization of function, using [StridedCopy](https://github.com/PaddlePaddle/Paddle/pull/4205/files) in context_project is also appropriate. But from the efficiency analysis, the efficiency of using StridedCopy is not high.
For example, the length of sequence is 20, context_length is 3, the realization of StridedCopy is equivalent to the replication of the 20*3 times, and for col_matrix, the address of each time replication is discontinuous.

**Pull request merged**